### DISCUSSION TITLE
Custom CSS indicator

### DISCUSSION THREAD

**Discussion by Systemedic** [elementor/elementor#22519] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Each element can have custom CSS and it can be hard to determine where you used custom css.

### Describe the solution you'd like

When custom CSS is used an icon will be placed next to the elementor in the navigator so it is clear that that element uses custom CSS.

**Comment by loucasl** [comment]
This already exists

https://elementor.com/help/navigator-indicators/

**Comment by Systemedic** [comment]
Never knew that. I only see one down side with this way of implementation: you have to do a hover to see what custom thing is used.

**Reply by jazir555** [reply]
Basically the implementation just needs a more concise/obvious icon or some other variation?

**Reply by Systemedic** [reply]
Correct. Making use of color or a small tag would be better in my opinion.

**Comment by stefanradu9** [comment]
have a look here https://www.wordpresstoday.agency/2024/12/22/add-custom-css-indicator-to-elementor-navigator-elements/
